Keith Kennedy

Kennedykid1995 opened the pull request:
I just started on the objects.js creating the objects for interns

mixelpixel committed:
Stretch Task header

- match header style for Stretch tasks from last week's curriculum.
- console.logging assignment decription

mixelpixel committed:
minor copy edits

- line 14 last sentence
- line 22 period

mixelpixel committed:
minor

- Kenna >>> Kennan

BigKnell committed:
Merge branch 'master' of https://github.com/LambdaSchool/JavaScript-I

BigKnell committed:
Merge pull request #2 from LambdaSchool/mixelpixel-patch-2

PATRICK KENNEDY

BigKnell committed:
Merge pull request #1 from LambdaSchool/mixelpixel-patch-1

Patrick Kennedy

Kennedykid1995 committed:
Starting Javascript 1

I just started on the objects.js creating the objects for interns

Kennedykid1995 committed:
Object.js finished

I have finished the object.js, had some trouble on the last question before the stretch, not sure if I did it right cause I did get it to multiply, but I did not put Antonietta.math(3, 4), just put math(3, 4)

Kennedykid1995 committed:
Started Arrays.js

I dont really understand this section, just cause really am not sure on writing for loops

Kennedykid1995 committed:
Array.js working still

I have gotten challenge 4 done as well as challenge 2

TERR1E commented on assignments/arrays.js:
```diff
@@ -63,14 +63,14 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a 2011 "jeep" "Wrangler"` );
```
The answer is technically correct but you were supposed to reference it using code.
1. Which code do you use to get a specific index value out of an array?
2. Which code do you use to get a specific value out of the property of an object?

The same goes for Challenge 2 below.

TERR1E commented on assignments/arrays.js:
```diff
@@ -63,14 +63,14 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+console.log(`Last car is a "Lincoln" "Town Car"` )
```
Which code would you use to get the last index value of an array?

TERR1E commented on assignments/arrays.js:
```diff
@@ -80,17 +80,19 @@ console.log();
+  carYears.push(inventory[i].car_year);
+}
+console.log(carYears);
 console.log();
```
You don't need to put an empty console.log

Kennedykid1995 committed:
Not done yet

Made some changes to car array assingment

TERR1E commented on assignments/objects.js:
```diff
@@ -18,31 +18,66 @@ let example = {
+function math(x, y){
+  return x*y;
+}
+console.log(math(3,4));
```
Check out the hint they've given to you:
`console.log(antonietta.multiplyNums(3,4));`

1. How would you name your function based on the hint?
2. What are the ways of assigning a new property to the existing object (Antonietta)?

Check out the links below:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Working_with_Objects

https://stackoverflow.com/questions/1184123/is-it-possible-to-add-dynamically-named-properties-to-javascript-object

http://www.dyn-web.com/tutorials/object-literal/properties.php

Kennedykid1995 committed:
Working on the Array.js still

added in my code, it doesn't work I am still trying to figure it out, the Alphabetical, <2000, and BMW and Audi

Kennedykid1995 committed:
Finished Alphabet Challenge 3

I completed the Alphabet Challenge and now I have two more to complete, but Im gonna work on the Assingment 2 before finishing up this one
